h/services/search_index/_queue.py

```diff
@@ -44,11 +46,28 @@ def add_all(self, annotation_ids, tag, schedule_in=None):
                 name="sync_annotation",
                 scheduled_at=scheduled_at,
                 priority=priority,
-                kwargs={"annotation_id": annotation_id},
+                kwargs={"annotation_id": URLSafeUUID.url_safe_to_hex(annotation_id)},
             )
             for annotation_id in annotation_ids
         )
 
+    def add_annotations_between_times(self, start_time, end_time, tag):
```

Comment:
The docstring for this method is on the corresponding `SearchIndexService.add_annotations_between_times()` since that's public and this is private